client/src/components/Feed.jsx

```diff
@@ -15,18 +15,22 @@ class Feed extends React.Component {
 
     return (
       <div>
-        {!hasTransactions
-          ? <div> No transactions available. </div>
-          : <List>
-              {this.props.transactions.items && this.props.transactions.items.map((transaction, i) => {
-                let isLastItem = i === this.props.transactions.items.length - 1;
-                return (
-                  <FeedTransaction key={transaction.transactionId} isLastItem={isLastItem} transaction={transaction} />
-                );
-              })}
-            </List>
-        }
-        {showMoreButton && <button onClick={ () => this.props.loadMoreFeed(this.props.type, this.props.userId) } >Show More</button>}
+        <div>
+          {!hasTransactions
+            ? <div> No transactions available. </div>
+            : <List>
+                {this.props.transactions.items && this.props.transactions.items.map((transaction, i) => {
+                  let isLastItem = i === this.props.transactions.items.length - 1;
+                  return (
+                    <FeedTransaction key={transaction.transactionId} isLastItem={isLastItem} transaction={transaction} />
+                  );
+                })}
+              </List>
+          }
+        </div>
+        <div className='show-more'>
+          {showMoreButton && <button className='btn' onClick={ () => this.props.loadMoreFeed(this.props.type, this.props.userId) } >Show More</button>}
+        </div>
       </div>
     );
```

Comment:
@acenine - This added an empty div inside a div. Is there a reason for nesting this in an empty div?